SourceX/storm/storm.cpp

```diff
@@ -309,15 +309,19 @@ HWND SDrawGetFrameWindow(HWND *sdraw_framewindow)
 void *SMemAlloc(unsigned int amount, char *logfilename, int logline, int defaultValue)
 {
 	// fprintf(stderr, "%s: %d (%s:%d)\n", __FUNCTION__, amount, logfilename, logline);
-	assert(amount != -1u);
-	return malloc(amount);
+	//assert(amount != -1u);

```

Comment:
Are the changes to SMemAlloc needed and would this be the correct implementation?

Reply:
It was disabled for testing only.